app-office/fcrm/fcrm-0.2.0~beta.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Customer Relations Management for Freelancers and Individual Contractors"
+DESCRIPTION="The application is dead simple, with just the complexity and feature
+ a high value Freelancer needs to do a brilliant job with sales and customer
+ relations.
+Features:
+* Contact management (companies or private persons).
+* Contact-person management at the companies.
+* Intents (pipelines) - some mini-projects with clear defined goals that you want to achieve.
+  For example , sell something to, or get the attention from, a prospect.
+* Actions - Steps / tasks to perform to move an intent forward against completion.
+  For example - send a follow-up mail at a specific date.
+* Document management documents and mails are linked to customers, persons, intents or actions.
+* Journal - a list of all the relevant things that has happened within the relation with a contact.
+  This is updated automatically when you add or change information.
+* Data is stored locally in a sqlite database."
+COPYRIGHT="1999-2019 Jarle Aase"
+HOMEPAGE="https://github.com/jgaa/f-crm"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jgaa/f-crm/archive/v${portVersion/\~beta/-beta}.tar.gz"
+CHECKSUM_SHA256="ac404cdcf7f5413f744a780299717790397bfc59017cbb043997d66b5af63d10"
+SOURCE_FILENAME="f-crm-${portVersion/\~beta/-beta}.tar.gz"
+SOURCE_DIR="f-crm-${portVersion/\~beta/-beta}"
+ADDITIONAL_FILES="fcrm.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
```

Comment:
x86_gcc2 can't be build due to requirements, suggestion:
```diff
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Reply:
ok.done thanks